install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -188,8 +190,9 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   severity: '"Info"'
-  timestamp: request.time
+  timestamp: request.time | timestamp("2017-01-01T00:00:00Z")
```

Comment:
can we avoid this fallback timestamp in any way?  I think for http requests, we will always have `request.time` and `response.time`.

Reply:
Couple of options:
1) Add conditionalTimestamp in externs.go which would default to request.time for http and grpc and this unix epoch time for tcp
2) Separate out accesslog entry for tcp and http/grpc like we have for metrics

I think second option would correspond to what we have for metrics too.. what do you think?

Reply:
Yeah, we should update the current access log rule to make it only dispatch for http request.

Reply:
Thanks Separated tcp and http accesslog entries

Reply:
Also, added connection.event as a variable in the log for tcp as compared to filtering on open and close events only..

Reply:
This change is missing from version 1.0.6. Please comment. My side car does not standup when deploying ElasticSearch.

Wanted to understand if the issue has crept in again.

Thank you.

Sundip